rspace-bench/src/test/scala/coop/rchain/rspace/bench/EvalBenchStateBase.scala

```diff
@@ -26,6 +29,11 @@ trait EvalBenchStateBase {
   def doSetup(): Unit = {
     deleteOldStorage(dbDir)
 
+    implicit val scheduler: Scheduler = monix.execution.Scheduler.Implicits.global
+
+    runtime.reducer.setAvailablePhlos(Cost(Integer.MAX_VALUE)).runSyncUnsafe(1.second)
+    runtime.replayReducer.setAvailablePhlos(Cost(Integer.MAX_VALUE)).runSyncUnsafe(1.second)
```

Comment:
we have in code `TaskContrib.unsafeRunSync` maybe it would make sense to use it?

Reply:
It exactly suits my needs, updated also WideBenchBaseState, since it also used `.runSyncUnsafe(1.second)` approach.